src/jimgw/jim.py

```diff
@@ -172,7 +171,7 @@ def sample(
                     )
                 )[0]
 
-                key, subkey = jax.random.split(key)
+                rng_key, subkey = jax.random.split(subkey)
```

Comment:
This is a minor issue, but I guess you meant to use `rng_key` instead of `subkey`.
```suggestion
                rng_key, subkey = jax.random.split(rng_key)
```